openedx/features/course_search/static/course_search/templates/course_search_item.underscore

```diff
@@ -1,4 +1,7 @@
-<div class="result-excerpt"><%= HtmlUtils.ensureHtml(excerptHtml) %></div>
-<a class="result-link" href="<%- url %>"><%- gettext("View") %> <span class="icon fa fa-arrow-right" aria-hidden="true"></span></a>
-<span class="result-location"><%- location.join(' ▸ ') %></span>
-<span class="result-type"><%- content_type %></span>
+<div class="result-type">
+    <i class="fa fa-<%- unit_icon %>"></i>
+</div>
+<div class="result-info">
+    <a class="result-link" href="<%- url %>"><%- location.join(' ▸ ') %></a>
+    <div class="result-excerpt"><%= HtmlUtils.ensureHtml(excerptHtml) %></div>
```

Comment:
In cases where there is no added excerpt to display, we were seeing lots of vertical whitespace in the screenshots.
Can we have this extra whitespace condensed/removed, when there is no text to display?

Ideally, this would just automatically collapse via CSS when there's nothing to display, but if we need to force the issue, maybe we need to just not add this `result-excerpt` `DIV` if the text is empty? Maybe try the former first?

Reply:
Used css to fix that issue. ``result-excerpt`` was with a display set to ``inline-block``. Moving that to ``block`` fixed the issue.

Reply:
:+1: